scipy/spatial/distance.py

```diff
@@ -144,6 +146,67 @@ def _validate_vector(u, dtype=None):
         raise ValueError("Input vector should be 1-D.")
     return u
 
+def directed_hausdorff(u, v):
+    """
+    Computes the directed Hausdorff distance between N-dimensional
+    point sets using a Euclidean metric.
+    
+    .. versionadded:: 0.19.0
+
+    Parameters
+    ----------
+    u : (M,N) ndarray
+        Input array.
+    v : (O,N) ndarray
+        Input array.
+
+    Returns
+    -------
+    d : double
+        The directed Hausdorff distance between arrays `u` and `v`.
+
+    Notes
+    ----------
+    Uses the early break technique and the random sampling approach described
+    by [Taha2015]_. Although worst-case performance is polynomial (as with the
+    brute force algorithm), this is exceedingly unlikely in practice, and
+    almost-linear time complexity performance can normally be expected for the
+    average case.
+
+    References
+    ----------
+    
+    .. [Taha2015] Taha and Hanbury (2015) An efficient algorithm
+                          for calculating the exact Hausdorff distance.
+                          IEEE Transactions On Pattern Analysis And
+                          Machine Intelligence 37: 2153-63. 
```

Comment:
Can you reformat this into the format given in the References section of https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt?